js/interface.js

```diff
@@ -1084,6 +1084,17 @@ function createFolder() {
       return;
     }
 
+    if (result.length > 45) {
+      Fliplet.Modal.alert({ 
+        title: 'Failed to create folder',
+        message: 'Folder name must be 45 characters or less'
+      }).then(function () {
+        createFolder()
```

Comment:
Pass the result back to the function so that the user doesn't lose the folder name they just entered. You'll need to adjust how `createFolder()` works so it takes the parameter and uses it for the prompt value.

```suggestion
        createFolder(result);
```

Also, missing semicolon. Please add ESLint.

Reply:
fixed